packages/editor-ui/src/mixins/expressionManager.ts

```diff
@@ -83,14 +85,19 @@ export const expressionManager = defineComponent({
 				throw new Error(`Failed to parse expression: ${this.editor.state.doc.toString()}`);
 			}
 
+			const customSkipSegments = 'skipSegments' in this ? (this.skipSegments as string[]) : [];
```

Comment:
Uncertain if we should be relying on a property `skipSegments` that is neither defined in the mixin nor explicitly passed to it.